apps/analysis_framework/serializers.py

```diff
@@ -709,3 +709,42 @@ def create(self, validated_data):
         validated_data['framework'] = self.framework
         validated_data['added_by'] = self.context['request'].user
         return super().create(validated_data)
+
+
+class AnalysisFrameworkCloneSerializer(serializers.Serializer):
+    af_id = IntegerIDField(required=True)
+    title = serializers.CharField(required=True)
+    description = serializers.CharField(required=False)
+    project = serializers.PrimaryKeyRelatedField(queryset=Project.objects.all(), required=False)
+
+    def validate_af_id(self, af_id):
+        af = (
+            AnalysisFramework
+            .get_for_gq(self.context['request'].user, only_member=False)
+            .filter(pk=af_id)
+            .first()
+        )
+        if af is None:
+            raise serializers.ValidationError('Analysis Framework does not exist')
+        if not af.can_clone(self.context['request'].user):
+            raise serializers.ValidationError(
+                'User does not have permission to modify the AF'
+            )
+        return af
+
+    def validate_project(self, project):
+        if not project.can_modify(self.context['request'].user):
+            raise serializers.ValidationError('User does not have permission to modify the project')
+        return project
+
+    def create(self, validated_data):
+        # NOTE: af is an obj, check validate_af_id
+        af = validated_data['af_id']
+        new_af = af.clone(self.context['request'].user, validated_data['title'], validated_data.get('description'))
```

Comment:
Let's use keyword for keyword arguments
```suggestion
        new_af = af.clone(self.context['request'].user, title=validated_data['title'], description=validated_data.get('description'))
```